test/parallel/test-no-harmony-top-level-await.mjs

```diff
@@ -0,0 +1,8 @@
+// Flags: --no-harmony-top-level-await
```

Comment:
@devsnek this test passes even without the flag. Not sure how to properly test what you were requesting for though :(

Reply:
@MylesBorins it should pass with and without the flag, so that's good :)